tutorials/asset_pipeline/managing_image_files.rst

```diff
@@ -111,3 +111,34 @@ Atlas for a set of images. It is also possible to ask the exporter to
 scale all images (or selected groups).
 
 More information on the :ref:`doc_exporting_images` page.
+
+Fixing PNGs
```

Comment:
IMO the header title is too broad. Something like _"Fixing PNGs iCC profile"_ could be better.